test/presenters/base.presenter.js

```diff
@@ -0,0 +1,36 @@
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const BasePresenter = require('../../app/presenters/base.presenter')
+
+describe('Base presenter', () => {
+  afterEach(async () => {
+    Sinon.restore()
+  })
+
+  it('presents data', async () => {
+    // Stub _presentations to simulate a child class with presentation properties
+    Sinon.stub(BasePresenter.prototype, '_presentations').callsFake((data) => {
+      return { after: data.before }
+    })
+
+    const testData = { before: true }
+    const testPresenter = new BasePresenter(testData)
+    const presentation = testPresenter.call()
+
+    expect(presentation.after).to.equal(true)
+  })
+
+  it('throws an error if translations are not specified', async () => {
```

Comment:
```suggestion
  it('throws an error if presentations are not specified', async () => {
```

Though bear in mind my previous comment about making presentations singular.